app/views/admin/dashboard/show.html.erb

```diff
@@ -108,7 +108,7 @@
 
         <tr class="table-light">
 
-          <td class="row-title"><strong><%= facility.name %></strong> <small style="display: block;">(<%= number_to_percentage((@control_rate_by_facility[facility.id] || 0) * 100, precision: 1) || 'N/A' %> control)</small></td>
+          <td class="row-title"><strong><%= facility.name %></strong> <!--small style="display: block;">(<%= number_to_percentage((@control_rate_by_facility[facility.id] || 0) * 100, precision: 1) || 'N/A' %> control)</small--></td>
```

Comment:
Let's just delete this code instead of commenting it out. Git takes care of remembering our changes 🙌